.circleci/config.yml

```diff
@@ -714,9 +714,27 @@ jobs:
       - build/determinator:
           <<: *federated_build_verify
       - run:
+          name: generate test certs and snowflake
+          command: |
+            # TODO add rootCA.pem and snowflake files in the ubuntu-1604:201903-01 image
+            # create directories
+            cd ${MAGMA_ROOT} && mkdir -p .cache/test_certs/ && mkdir -p .cache/feg/
```

Comment:
I would use  `mkdir ${MAGMA_ROOT}/.cache/test_certs/` && `mkdir ${MAGMA_ROOT}.cache/feg/`and remove the cd.

Reply:
ohh I forgot I had  automerge. 

Will create a pr with the changes